frontend/src/pages/surveys/styles.module.scss

```diff
@@ -0,0 +1,31 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 80px;
+  height: 100vh;
+  background-image: url("/images/survey-1440x1024.png");
```

Comment:
remove this image from public and use auth-background from assets/images

Reply:
Done